features/bevragen/protocollering.feature

```diff
@@ -0,0 +1,85 @@
+#language: nl
+
+Functionaliteit: Als burger wil ik zien wie welke gegegevens van mij heeft gezien of gebruikt
+  Zoeken en raadplegen van gegevens van burgers worden "geprotocolleerd" (formeel gelogd).
+
+  Protocollering moet gebeuren op twee niveau's:
+  - bij de RvIG op het niveau van afnemende organisaties
+  - bij de afnemende organisatie op het niveau van gebruikers
+
+  Deze feature beschrijft alleen de werking van protocollering op het niveau van afnemende organisaties bij de bron (RvIG)
+
+  Daarin worden de volgende gegevens vastgelegd:
+  - request_id: unieke identificatie van de berichtuitwisseling waarmee het protocolleringsrecord te koppelen is aan andere logging van de API
+  - request_datum: tijdstip waarop de protocollering is vastgelegd
+  - afnemer_code: identificatiecode van de afnemende organisatie, zoals die is opgenomen in de gestuurde Oauth 2 token
+  - anummer: het A-nummer van de persoon waarvan gegevens zijn gevraagd of geleverd
```

Comment:
Waarom kies je er hier voor om het a-nummer vast te leggen i.p.v. het burgerservicenummer?

Reply:
Ik realiseer me nu dat dit waarschijnlijk geen keuze van jou is maar een gegeven waar we ons bij aan moet sluiten.